kythe/proto/storage.proto

```diff
@@ -203,3 +204,10 @@ message VNameRewriteRule {
 message VNameRewriteRules {
   repeated VNameRewriteRule rule = 1;
 }
+
+// RepoMetadata provides storage for additional information about a specific
+// build from a repository. This message is stored in the
+// CompilationUnit.details field.
+message BuildMetadata {
+  google.protobuf.Timestamp commit_timestamp = 1;
```

Comment:
Is it clear what this is a "commit" of? What if the build comes from multiple repositories? Should we be more generic, and use something like "origin_timestamp"?

Reply:
`origin_timestamp` sounds fine to me. Or we can clarify that it's the commit timestamp of the "superproject"?